engines/sherlock/image_file.cpp

```diff
@@ -129,7 +145,13 @@ void ImageFile::load(Common::SeekableReadStream &stream, bool skipPalette, bool
 
 		frame._pos = stream.pos();
 
-		if (_name.empty()) {
+		if (invalid) {
+			frame._decoded = true;
+			frame._frame.create(frame._width, frame._height, Graphics::PixelFormat::createFormatCLUT8());
+			byte *dest = (byte *)frame._frame.getPixels();
+			Common::fill(dest, dest + frame._width * frame._height, 0xff);
```

Comment:
That will not work—either use `frame._frame.pitch` instead of `frame._width` or better  use`frame._frame.fillRect()`.

The surface pitch may differ from the width.

Reply:
Done